tests/urlpatterns/tests.py

```diff
@@ -308,6 +308,14 @@ def test_allows_non_ascii_but_valid_identifiers(self):
         match = p.resolve('hello/1/')
         self.assertEqual(match.kwargs, {'\u0394': '1'})
 
+    def test_name_is_passed_as_kwarg(self):
+        msg = (
+            f'Keyword arguments must be dict, not str. '
+            f'Hint: You are probably trying to pass the URL name as a positional argument.'
+        )
+        with self.assertRaisesMessage(TypeError, msg):
+            path('hello/', lambda r: None, 'foo')
```

Comment:
You can reuse `empty_view`:
```suggestion
            path('hello/', empty_view, 'name')
```

Reply:
Yes, thanks!